src/main/webapp/js/finscholar/singlepageview.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @fileoverview This class renders view for scholarship. */
+
+goog.module('finscholar.singlepageview');
+
+/**
+ * Class for single scholarship/college page view.
+ * @public
+ */
+class SinglePageView {
+
+  /**
+   * @constructor
+   */
+  constructor(template) {
+    /** 
+     * @private @const {?PageDataHandler} dataHandler_ 
+     * The object fetches and formats scholarship data.
+     */
+    this.dataHandler_ = null;
+    /**
+     * @private @const {!function(PageObject):Element}
+     */
+    this.template_ = template;
+  }
+
+  /** @param {!PageDataHandler} dataHandler */
+  setDataHandler(dataHandler) {
+    this.dataHandler_ = dataHandler;
+  }
```

Comment:
Why is this needed? You have it as a const above. This probably prevents your code from compiling.